apps/io-wallet-user-func/src/infra/azure/cosmos/wallet-instance.ts

```diff
@@ -81,7 +84,10 @@ export class CosmosDbWalletInstanceRepository
     );
   }
 
-  getAllByUserId(userId: WalletInstance["userId"]) {
+  getLastActiveWalletInstanceByUserId(
```

Comment:
it's not the last one, but it's the one different from the input `walletInstanceId`